tensorflow/contrib/tensor_forest/kernels/v4/split_collection_operators.cc

```diff
@@ -136,6 +136,33 @@ void SplitCollectionOperator::CreateAndInitializeCandidateWithExample(
   stats_.at(node_id)->AddSplit(split, input_data, target, example);
 }
 
+void SplitCollectionOperator::CreateAndInitializeCandidateWithExample(
+    const std::unique_ptr<TensorDataSet>& input_data, const InputTarget* target,
+    int example, int rand_feature, int32 node_id) const {
+  decision_trees::BinaryNode split;
+  float bias;
+  int type;
+  decision_trees::FeatureId feature_id;
+  input_data->RandomSample(example, &feature_id, &bias, &type, rand_feature);
```

Comment:
hm wouldn't that feature_id=rand_feature?
Actually looks like not - it seems that the feature id is the relative to the number of features presentin an instance.... Ok